src/tests/playwright/pages/dsp-graph-evaluation.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script type="module" defer>
+      import { evaluateGraph } from './util/audio-comparer.js';
+      import { test, evaluate } from './util/audit.js';
+
+      // My Render Length (seconds)
+      // noinspection JSConstantReassignment,JSValidateTypes
+      test((async () => {
+        const length = 1;
+        // My DSP Graph (function)
+        const myGraph = ctx => {
```

Comment:
The convention I like is spelling out things when possible. Hence, `context` or `audioContext`.